Curve-finance-suite/src/agents/curve.gauge.set.rewards.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import Web3 from "web3";
+import abi from "../utils/curve-gauge";
+
+// @ts-ignore
+import abiDecoder from "abi-decoder";
+abiDecoder.addABI(abi);
+
+export const web3 = new Web3();
+
+export const setRewards = {
+  name: "set_rewards",
+  outputs: [],
+  inputs: [
+    { type: "address", name: "_reward_contract" },
+    { type: "bytes32", name: "_sigs" },
+    { type: "address[8]", name: "_reward_tokens" },
+  ],
+  stateMutability: "nonpayable",
+  type: "function",
+  gas: 2304194,
+};
+
+const createFinding = (alertId: string): Finding => {
+  return Finding.fromObject({
+    name: "Set Rewards funciton called",
+    description: "Set Rewards funciton called on pool",
```

Comment:
on what pool? Is the address field in metadata the pool? Can you add the pool in the metadata? Can you add any metadata to make this finding more informative?